custom_components/samsungtv_tizen/const.py

```diff
@@ -69,12 +70,40 @@ class PowerOnMethod(Enum):
 
 STD_APP_LIST = {
     # app_id: smartthings app id (if different and available)
-    "org.tizen.browser": "",                    # Internet
-    "11101200001": "RN1MCdNq8t.Netflix",        # Netflix
+    "org.tizen.browser": "",  # Internet
+    "11101200001": "RN1MCdNq8t.Netflix",  # Netflix
     "111299001912": "9Ur5IzDKqV.TizenYouTube",  # YouTube
-    "3201512006785": "org.tizen.ignition",      # Prime Video
-    "3201901017640": "MCmYXNxgcu.DisneyPlus",   # Disney+
-    "11091000000": "4ovn894vo9.Facebook",       # Facebook
-    "3201601007250": "QizQxC7CUf.PlayMovies",   # Google Play
-    "3201606009684": "rJeHak5zRg.Spotify",      # Spotify
+    "3201512006785": "org.tizen.ignition",  # Prime Video
+    "3201901017640": "MCmYXNxgcu.DisneyPlus",  # Disney+
+    "11091000000": "4ovn894vo9.Facebook",  # Facebook
+    "3201601007250": "QizQxC7CUf.PlayMovies",  # Google Play
+    "3201606009684": "rJeHak5zRg.Spotify",  # Spotify
 }
+
+# Logo feature constants
```

Comment:
I wonder if make sense define this constant here or if is better to leave them in the dedicated class `logo.py` and directly import from there.

Reply:
Moved the relevant constants all to logo.py. I kept CONF_LOGO_OPTION as part of all the CONF_ variables in const.py. I think it makes sense to keep that one there as it ties in with all the other core configuration options of the component.

Honestly, I'm fine either way... I can think of valid arguments for both approaches.